examples/validation-app/README.md

```diff
@@ -0,0 +1,126 @@
+# @loopback/example-validation-app
+
+An example application to demonstrate validation in LoopBack.
+
+## Summary
+
+This application shows how to add validation in a LoopBack application. It
+exposes `/coffee-shops` endpoints to create/read/update/delete a `CoffeeShop`
+instance with the in-memory storage.
+
+### Key artifacts
+
+1. [CoffeeShop model](src/models/coffee-shop.model.ts): Shows how to add
+   validation using [AJV](https://www.npmjs.com/package/ajv).
+
+   Example to limit lenght on a string:
+
+   ```ts
+   @property({
+    type: 'string',
+    required: true,
+    // --- add jsonSchema -----
+    jsonSchema: {
+      maxLength: 10,
+      minLength: 1,
+    },
+    // ------------------------
+   })
+   city: string;
+   ```
+
+2. [ValidatePhoneNumInterceptor](src/interceptors/validate-phone-num.interceptor.ts)
+   and [CoffeeShopController](src/controllers/coffee-shop.controller.ts): Shows
+   how to add reusable validation logic in interceptors
```

Comment:
The comment here seems to apply to the interceptor rather than the controller

Reply:
yes. maybe i should split into 2 bullets.